andi/andi.py

```diff
@@ -24,15 +26,19 @@ def inspect(func: Callable) -> Dict[str, List[Optional[Type]]]:
     return res
 
 
+TypeContainerOrCallable = Union[
+    Container[Type],
+    Callable[[Optional[Type]], bool]
```

Comment:
is Optional needed here, as direct None support seems to be removed?

Reply:
Yes, done in https://github.com/scrapinghub/andi/pull/8/commits/726a3f37cfa2bd77890e1f8b748928825d27e25b